buildscripts/mayactl/build.sh

```diff
@@ -6,7 +6,7 @@ set -e
 # Get the parent directory of where this script is.
 SOURCE="${BASH_SOURCE[0]}"
 while [ -h "$SOURCE" ] ; do SOURCE="$(readlink "$SOURCE")"; done
-DIR="$( cd -P "$( dirname "$SOURCE" )/../../cmd/mayactl" && pwd )"
+DIR="$( cd -P "$( dirname "$SOURCE" )/../../" && pwd )"
```

Comment:
Is this the change, that is forcing us to add ./cmd/mayactl in the below lines.

Reply:
No , above was added by me , then removed